azext_iot/tests/settings.py

```diff
@@ -15,7 +15,19 @@
     "azext_iot_testhub",
     "azext_iot_teststorageaccount",
     "azext_iot_teststoragecontainer",
-    "azext_iot_rbac_max_tries"
+    "azext_iot_rbac_max_tries",
+    "azext_iot_ep_rg",
+    "azext_iot_eventhub_namespace",
+    "azext_iot_eventhub_instance",
+    "azext_iot_eventhub_policy",
+    "azext_iot_servicebus_namespace",
+    "azext_iot_servicebus_queue",
+    "azext_iot_servicebus_topic",
+    "azext_iot_servicebus_policy",
+    "azext_iot_cosmos_namespace",
+    "azext_iot_cosmos_database",
+    "azext_iot_cosmos_collection",
+    "azext_iot_user_identity"
```

Comment:
Do these need added to [`set-testenv-sentinel.yml`](https://github.com/Azure/azure-iot-cli-extension/blob/dev/.azure-devops/templates/set-testenv-sentinel.yml)?

Reply:
![image](https://user-images.githubusercontent.com/73560279/208191372-03e93961-41d8-4a12-9a4c-34273e919789.png)

Also I dont even use these so I will just remove them. This is what style of test variables I wanted to go for originally

Reply:
also my changes do not affect the smoke test so anything needed for the smoke test is still needed